rasa/core/tracker_store.py

```diff
@@ -63,9 +66,15 @@
 # default value for key prefix in RedisTrackerStore
 DEFAULT_REDIS_TRACKER_STORE_KEY_PREFIX = "tracker:"
 
+CONNECTION_ERRORS = (
+    NoCredentialsError,
```

Comment:
it might make sense to ignore more errors from `botocore` (see [all the errors here](https://github.com/boto/botocore/blob/develop/botocore/exceptions.py))

Reply:
@wochinge I dunno if you saw this. Any opinion?

Reply:
It could make sense to do that indeed. I still have a concern that if we're ignoring all `BotoCoreErrors` in Sentry, we'll ignore both errors generated by users (e.g. wrong connection URIs for instance, which is good) and errors generated by rasa (e.g. some parameters that we send as part of the TrackerStore do not pass validation) 🤔

Reply:
🤔 but I think it's better that way. And it's not like errors are gonna be silent, users will still see them.